example/passthrough_hp.cc

```diff
@@ -20,7 +20,8 @@
  *
  * If --nocache is specified, the source directory may be changed
  * directly even while mounted and the filesystem will continue
- * to work correctly.
+ * to have correct file attributes. For data coherency --direct-io is
```

Comment:
Could you expand on this? It's not clear to me what's meant by that (alternatively, omitting it is also fine).

Reply:
Hmm, what I mean is that --nocache just disables the attribute cache, but still data inconsistencies might come up.